extensions/gubbins/gubbins-db/src/gubbins/db/sql/jobs/db.py

```diff
@@ -40,7 +40,7 @@ async def getJobJDL(  # type: ignore[override]
         info = (await self.conn.execute(stmt)).scalar_one()
         return {"JDL": jdl, "Info": info}
 
-    async def setJobAttributes(self, job_id, jobData):
+    async def setJobAttributesBulk(self, jobData):
```

Comment:
FYI, this is not yet enforced, but once https://github.com/DIRACGrid/diracx/pull/334 will be merged, we will have to use snake case for function/method names.

Reply:
(I would like to merge this before it gets too out of sync again and we fall into a cycle of never merging it)

Reply:
Yes sure, it just to inform you, but yes I will adjust https://github.com/DIRACGrid/diracx/pull/334 once this PR is merged.